lib/pdk/validators/util/task_metadata_schema.rb

```diff
@@ -0,0 +1,65 @@
+module PDK
+  module Validate
+    module TaskMetadataSchema
+      SCHEMA = '{
```

Comment:
Please point here  to the canonical version of this schema, so that there can be no doubt about which copy is the primary.